src/main/java/de/chojo/repbot/commands/abuseprotection/AbuseProtection.java

```diff
@@ -24,16 +24,26 @@ public AbuseProtection(Guilds guilds) {
                         .handler(new Info(guilds)))
                 .subCommand(SubCommand.of("maxmessageage", "command.abuseprotection.maxmessageage.description")
                         .handler(new MaxMessageAge(guilds))
-                        .argument(Argument.integer("minutes", "command.abuseprotection.maxmessageage.minutes.description")))
+                        .argument(Argument.integer("minutes", "command.abuseprotection.maxmessageage.minutes.description")
+                                          .min(0)
+                                          .max(10_080)))
```

Comment:
Maybe add a comment explaining this magic number or compute it?

```suggestion
                                          .max(ChronoUnit.WEEKS.getDuration().toMinutes())))
```

Reply:
Thats actually a good idea